src/lib/mongoBackend/mongoNotifyContextAvailability.cpp

```diff
@@ -45,9 +45,7 @@ HttpStatusCode mongoNotifyContextAvailability
   const std::string&                   servicePath
 )
 {
-    // FIXME P10: we have been discussing about changing the "notifyFormat" name for something more neutral and more
-    // meaningfull for *both* subscription notification and cpr forwards
-    std::string notifyFormat = uriParam[URI_PARAM_NOTIFY_FORMAT];
+    const std::string notifyFormat = uriParam[URI_PARAM_NOTIFY_FORMAT];
```

Comment:
Ah, you are right: in this function is used to store in DB the format to use in forwards due to the registration (notifyContextAvailability has the same semantics that a registerContext).

NTC